setup.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/bash
```

Comment:
I assume, we should all run this script  to setup our dev environment. Could you add a short section to the README file to explain this?